### ISSUE TITLE
Add multiarch support

### ISSUE THREAD

**Issue by slamd64** [xelalexv/dregsy#43]
ARM64 architecture becomes very popular and AWS EKS/ECR already supports this. So, syncing AMD64+ARM64 is good idea.
https://docs.docker.com/docker-for-mac/multi-arch/

Skopeo is already supports this feature. So, it needs to bypass "--all" from Dregsy command line to Skopeo.

skopeo copy
options:
--all
If source-image refers to a list of images, instead of copying just the image which matches the current OS and architecture (subject to the use of the global --override-os, --override-arch and --override-variant options), attempt to copy all of the images in the list, and the list itself.

**Comment by slamd64** [comment]
Hi!

Do you have any plans for multiarch? :)

**Comment by xelalexv** [comment]
I think this would be a really useful enhancement. Problem is, I really need to get the filtering stuff out first. Unfortunately I don't have that much time to work on dregsy, so I can't promise when this is going to happen.

**Comment by xelalexv** [comment]
I finally got around to working on this. When syncing from a multi-platform source image, you will have two options now - select one platform image to sync, or sync all available platform images. This can be defined per mapping, for example:

```yaml
  mappings:
  - from: library/busybox
    to: library/busybox-arm64
    tags: ['latest']
    platform: linux/arm64/v8
```

This syncs the `linux/arm64/v8` from a *busybox* multi-platform image. To select all platforms, specify `platform: all`. `all` however only works with the *Skopeo* relay. If you want to sync a selection of platform images from the same multi-platform source image, several mappings with according `platform` settings can be defined. You need to be careful however not to map them into the same destination, i.e. use different `to` settings. Otherwise, the synced platform images will "overwrite" each other, with only the last image synced being available from the target repository. For example:

```yaml
  mappings:
  - from: library/busybox
    to: library/busybox-arm64
    tags: ['latest']
    platform: linux/arm64/v8
  - from: library/busybox
    to: library/busybox-amd64
    tags: ['latest']
    platform: linux/amd64
```

**Comment by xelalexv** [comment]
Implemented with PR #43.
